TheWarWithin/Priorities/DeathKnightFrost.simc

```diff
@@ -122,6 +125,7 @@ actions.breath+=/soul_reaper,if=fight_remains>5&target.time_to_pct_35<5&target.t
 actions.breath+=/howling_blast,if=(variable.rime_buffs|!buff.killing_machine.up&buff.pillar_of_frost.up&talent.obliteration&!buff.bonegrinder_frost.up)&runic_power>(variable.breath_rime_rp_threshold-(talent.rage_of_the_frozen_champion*6))|!dot.frost_fever.ticking
 actions.breath+=/horn_of_winter,if=rune<2&runic_power.deficit>30&(!buff.empower_rune_weapon.up|runic_power<variable.breath_rp_cost*2*gcd.max)
 actions.breath+=/obliterate,cycle_targets=1,if=buff.killing_machine.up|runic_power.deficit>20
+actions.breath+=/soul_reaper,if=fight_remains>5&target.time_to_pct_35<5&target.time_to_pct_0>5&active_enemies<=1&rune>2
```

Comment:
`target.time_to_pct_0` should be the same as `target.time_to_die` and the latter doesn't involve string-parsing to get the `0` out of `time_to_pct_0`.

`active_enemies<=1` will always be true because if you don't have at least one target, the addon will still pretend there is one.